### DISCUSSION TITLE
Is there any option in vvenc similar to AQ mode or AQ strength of x265?

### DISCUSSION THREAD

**Discussion by unuused** [fraunhoferhhi/vvenc#221] [Q&A]
I guess qpa have similar outcome for vvenc as aq mode.
But QPA only takes value of 1

**Comment by adamjw24** [comment]
What is aq mode?

**Reply by unuused** [reply]
adaptive quantization 
https://ieeexplore.ieee.org/abstract/document/6637940/

**Reply by adamjw24** [reply]
Yeah, that's quite similar. Except QPA is optimized for XPSNR. So the most important difference there is, it also has temporal weighting to calculate the QP offsets. QPA also takes value 0.

**Reply by unuused** [reply]
Yeah, just an idea.. wouldn't it be great if it qpa takes strength values to make it more or less agressive.

**Reply by adamjw24** [reply]
Well, I honestly think our QPA is very good just as it is. The SW is open-source tho, can't be to hard too add some weighting to the QP offsets.
